webapp-django/crashstats/settings/base.py

```diff
@@ -177,18 +176,33 @@ def path(*dirs):
 # Always generate a CSRF token for anonymous users.
 ANON_ALWAYS = True
 
-LOG_LEVEL = logging.INFO
+LOGGING_LEVEL = config('LOGGING_LEVEL', 'INFO')
 
-HAS_SYSLOG = True  # XXX needed??
-
-LOGGING_CONFIG = None
```

Comment:
I'm 95% sure that having this line caused the later `LOGGING` to get ignored because it's telling Django "Yo--we'll do all our own logging configuration".

https://docs.djangoproject.com/en/1.8/ref/settings/#std:setting-LOGGING_CONFIG

If that's correct, then we don't want this line.